scripts/setup-wasm.mjs

```diff
@@ -1,21 +1,33 @@
 import path from 'path'
 import { readFile, writeFile } from 'fs/promises'
-import { copy } from 'fs-extra'
+import { copy, move, pathExists } from 'fs-extra'
 ;(async function () {
-  let wasmDir = path.join(process.cwd(), 'packages/next-swc/crates/wasm')
-  let wasmTarget = 'nodejs'
-  let wasmPkg = JSON.parse(
-    await readFile(path.join(wasmDir, `pkg-${wasmTarget}/package.json`))
-  )
-  wasmPkg.name = `@next/swc-wasm-${wasmTarget}`
+  try {
+    let wasmDir = path.join(process.cwd(), 'packages/next-swc/crates/wasm')
+    let wasmTarget = 'nodejs'
 
-  await writeFile(
-    path.join(wasmDir, `pkg-${wasmTarget}/package.json`),
-    JSON.stringify(wasmPkg, null, 2)
-  )
+    // CI restores artifact at pkg-${wasmTarget}
+    // This only runs locally
+    let folderName = (await pathExists(path.join(wasmDir, 'pkg')))
+      ? 'pkg'
+      : `pkg-${wasmTarget}`
 
-  await copy(
-    path.join(wasmDir, `pkg-${wasmTarget}`),
-    path.join(process.cwd(), `node_modules/@next/swc-wasm-${wasmTarget}`)
-  )
+    let wasmPkg = JSON.parse(
+      await readFile(path.join(wasmDir, `${folderName}/package.json`))
+    )
+    wasmPkg.name = `@next/swc-wasm-${wasmTarget}`
+
+    await writeFile(
+      path.join(wasmDir, `${folderName}/package.json`),
+      JSON.stringify(wasmPkg, null, 2)
+    )
+
+    await move(
```

Comment:
Do we want to keep using `copy` in case `node_modules` needs to be re-installed and the wasm build is removed?

Reply:
Yeah, sounds good. Originally I was doing a copy to rename `pkg -> pkg-${wasmTarget}` and then moving that. I changed it to update `pkg` and thought `move` might still be okay because I've noticed my rust projects have been taking up a lot of disk space. Would probably be more annoying to have to rebuild though. Updated 👍